src/wirecloud/fiware/social_auth_backend.py

```diff
@@ -44,6 +44,7 @@
 
 
 FIWARE_LAB_IDM_SERVER = 'https://account.lab.fiware.org'
+FIWARE_CLOUD_SERVER = 'https://cloud.lab.fiware.org'
```

Comment:
Rename this constant to `FIWARE_LAB_CLOUD_SERVER` while maintaining the `FIWARE_CLOUD_SERVER` name for the Django setting.